In-house patch to remove jasmine.js from Horizon.  This JavaScript
library is only used for testing and not used by Horizon during normal
operation.

--- horizon-2014.2.1/horizon/site_urls.py.orig    2015-02-02 15:31:57.209242227 -0700
+++ horizon-2014.2.1/horizon/site_urls.py        2015-02-02 15:32:11.276168850 -0700
@@ -22,8 +22,6 @@ from django.conf.urls import patterns
 from django.conf.urls import url
 from django.views.generic import TemplateView  # noqa

-from horizon.test.jasmine import jasmine
-
 urlpatterns = patterns('horizon.views',
     url(r'^home/$', 'user_home', name='user_home')
 )
@@ -38,7 +36,3 @@ urlpatterns += patterns('',
         name="set_language"),
     url(r'^i18n/', include('django.conf.urls.i18n'))
 )
-
-if settings.DEBUG:
-    urlpatterns += patterns('',
-        url(r'^jasmine/(.*?)$', jasmine.dispatcher))

--- horizon-2014.2.1/openstack_dashboard/settings.py.orig 2015-02-02 15:33:37.928460972 -0700
+++ horizon-2014.2.1/openstack_dashboard/settings.py 2015-02-02 15:33:58.344285143 -0700
@@ -29,7 +29,6 @@ import xstatic.pkg.bootstrap_scss
 import xstatic.pkg.d3
 import xstatic.pkg.font_awesome
 import xstatic.pkg.hogan
-import xstatic.pkg.jasmine
 import xstatic.pkg.jquery
 import xstatic.pkg.jquery_migrate
 import xstatic.pkg.jquery_quicksearch
@@ -162,8 +161,6 @@ STATICFILES_DIRS = [
         xstatic.main.XStatic(xstatic.pkg.hogan).base_dir),
     ('horizon/lib/font-awesome',
         xstatic.main.XStatic(xstatic.pkg.font_awesome).base_dir),
-    ('horizon/lib/jasmine-1.3.1',
-        xstatic.main.XStatic(xstatic.pkg.jasmine).base_dir),
     ('horizon/lib/jquery',
         xstatic.main.XStatic(xstatic.pkg.jquery).base_dir),
     ('horizon/lib/jquery',
